tests/cryptomatte_tests.py

```diff
@@ -522,19 +527,27 @@ class CryptomatteSetup(unittest.TestCase):
     def setUp(self):
         self.output_file_name = os.path.join(tempfile.gettempdir(), "result.exr")
 
+
         ai.AiBegin()
-        ai.AiMsgSetConsoleFlags(ai.AI_LOG_NONE)
-        ai.AiMsgSetConsoleFlags(ai.AI_LOG_WARNINGS | ai.AI_LOG_ERRORS)
+        self.universe = ai.AiUniverse()
+
+        msg_flags = ai.AI_LOG_WARNINGS|ai.AI_LOG_ERRORS|ai.AI_LOG_PROGRESS
+        ai.AiMsgSetLogFileFlags(self.universe, msg_flags)
+        ai.AiMsgSetConsoleFlags(self.universe, msg_flags)
+        ai.AiMsgWarning("[Cryptomatte test suite] - Rendering with Python. ")
+
+        self.session = ai.AiRenderSession(self.universe, ai.AI_SESSION_BATCH)
```

Comment:
Is this the session that `simulate_ipr_refreshes` is using? If so, that won't properly simulate IPR refresh. For that you want `AI_SESSION_INTERACTIVE`